projects/igniteui-angular/src/lib/grids/grid-base.directive.ts

```diff
@@ -5843,7 +5843,8 @@ export abstract class IgxGridBaseDirective extends DisplayDensityBase implements
      * @hidden @internal
      */
     public navigateToAddedRow(event) {
-        // TO DO: add navigation logic
+        this.navigateTo(this.dataView.length - 1);
```

Comment:
New record will not necessarily be added to the end of the data view. Some features (sorting, filtering, etc.) may change the target row location. You will need to check the added record index after the pipes have been executed to find the correct location.